### ISSUE TITLE
In RichText, aria-multiline is hardcoded to true, even when disableLineBreaks is true

### ISSUE THREAD

**Issue by markjaquith** [WordPress/gutenberg#34311]
If `<RichText />` is used with `disableLineBreaks`, `aria-multiline` is still true. Probably should be the opposite of `disableLineBreaks`.

https://github.com/WordPress/gutenberg/blob/3f2593fbfd61aff7bdbc2d177994c3b562f92bcc/packages/block-editor/src/components/rich-text/index.js#L310
